content/22.vaccines.md

```diff
@@ -178,6 +178,33 @@ As of December 11, 2020, the United States FDA approved this vaccine under an em
+The main finding were safety of the vaccine, decreased anti-spike IgG responses at day 28 with age, but similar level of antibody titres after receiving a booster dosage [@doi:10.1016/S0140-6736(20)32466-1].
+
+
+Phase II/III clinical trial was carried out as 4 independent locations: in the UK  [@clinicaltrials:NCT04400838], Brasil [@clinicaltrials:NCT04536051] and South Africa  [@clinicaltrials:NCT04444674] and finally in the United States in August 2020 [@clinicaltrials:NCT04516746; @url:https://s3.amazonaws.com/ctr-med-7111/D8110C00001/52bec400-80f6-4c1b-8791-0483923d0867/c8070a4e-6a9d-46f9-8c32-cece903592b9/D8110C00001_CSP-v2.pdf]. 
+In September 2020, the global trials had to be shortly paused due to a serious adverse neurological reaction in some of the participants [@doi:10.1038/d41586-020-02633-6].
+
+The interim results of Phase III were announced in a press release on November 23, 2020 [@url:https://www.astrazeneca.com/content/astraz/media-centre/press-releases/2020/azd1222hlr.html] and published in Lancet on December 9, 2020 [@doi:10.1016/S0140-6736(20)32661-1; @url:https://www.astrazeneca.com/content/astraz/media-centre/press-releases/2020/azd1222-oxford-phase-iii-trials-interim-analysis-results-published-in-the-lancet.html]. 
+The studies were met with skepticism, due to multiple inonsistencies in their design, including the errorneous dosing regimen in which multiple of 11,636 participants received half of the initial dose eventually leading to increased vaccine efficacy (90% vs 62.1% for two full doses protocol), using multiple testing sites and different placebos (meningococcal conjugate vaccine or saline) [@url:https://www.businessinsider.com/astrazenecas-covid-19-vaccine-trials-criticised-2020-11; @doi:10.1136/bmj.m4777; @doi:10.1016/S0140-6736(20)32623-4].
+The research in the UK showed that even a single dosage offered significant reduction in symptomatic SARS-CoV2 positive cases in the UK [@doi:10.1101/2021.03.01.21252652]. 
+This was one of the reasons to delay the second inoculation (booster dosage) and administer it no sonner than 12 weeks [@doi:10.1136/bmj.m4968; @doi:10.1016/S0140-6736(21)00528-6; @doi:10.1016/S0140-6736(21)00432-3]
+
+The emergence use of the vaccine was approved in the UK on December 30, 2020 [@url:https://www.astrazeneca.com/media-centre/press-releases/2020/astrazenecas-covid-19-vaccine-authorised-in-uk.html], followed by authorization by European EMA on January 29, 2021 [@url:https://www.ema.europa.eu/en/news/ema-recommends-covid-19-vaccine-astrazeneca-authorisation-eu]. 
+
+Due to emergence of new coronavirus variants, in particular so called South African 501Y.V2 variant (B.1.351 lineage), and reports of growing inefficacy of the vaccine to prevent mild to moderate cases [@url:https://arstechnica.com/science/2021/02/scary-22-vaccine-efficacy-in-south-africa-comes-with-heaps-of-caveats/], South Africa has stopped AstraZeneca's vaccine rollout on February 9, 2021 [@doi:10.1136/bmj.n372].
```

Comment:
```suggestion
Due to emergence of new coronavirus variants, in particular the so called South African 501Y.V2 variant (B.1.351 lineage), and reports of growing inefficacy of the vaccine to prevent mild to moderate cases [@url:https://arstechnica.com/science/2021/02/scary-22-vaccine-efficacy-in-south-africa-comes-with-heaps-of-caveats/], South Africa has stopped AstraZeneca's vaccine rollout on February 9, 2021 [@doi:10.1136/bmj.n372].
```

Reply:
This might fit better in the global distribution section, but will leave it here for now